### PR TITLE
docs(Translation.md): English-Uzbek tech list

### PR DESCRIPTION
**Pull request by turdiyev**

### CODE REVIEW

File: TRANSLATION.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+| (un)controlled component | boshqaril(may)adigan komponent |
+| debugging | debag qilish |
+| DOM container | DOM-konteyner |
+| error | xatolik |
+| feature | imkoniyat |
+| development | dasturlash |
+| development mode | dasturlash rejimi |
+| framework | freymvork |
+| function component | funksiyali komponent |
+| hook | ho'k, ilgak |
```
Comment (avocadowastaken) [inline TRANSLATION.md]: I think it's better to use only "uzbeknglish" version

File: TRANSLATION.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+| array | massiv |
+| arrow function |  arrow funksiya, strelka funksiya  |
+| attribute | atribut |
+| batch | yangilash guruhi |
+| batching | guruhlash |
+| bug | bag, xatolik  |
+| bundler | bandler |
+| callback | kolbek |
+| camelCase | *camelCase* |
+| class component | sinf komponentasi, class komponent |
```
Comment (avocadowastaken) [inline TRANSLATION.md]: I thinks `class komponent` is enough

**Review by avocadowastaken — COMMENTED** [review]
Great 👍 But I have two suggestions.

File: TRANSLATION.md (hunk 2 above)
Comment (turdiyev) [reply TRANSLATION.md]: hmm

**Comment by turdiyev** [comment]
> Great  But U have two suggestions.

I think you mean that add russian version. Ok, i will add russian column.

File: TRANSLATION.md (hunk 1 above)
Comment (turdiyev) [reply TRANSLATION.md]: what do you mean? could you explain again

Comment (avocadowastaken) [reply TRANSLATION.md]: keep only ho’k

Comment (umidbekkarimov) [reply TRANSLATION.md]: keep only ho’k

**Comment by avocadowastaken** [comment]
> I think you mean that add russian version. Ok, i will add russian column.

Nope 

These two:
https://github.com/reactjs/uz.reactjs.org/pull/2#pullrequestreview-201921988
https://github.com/reactjs/uz.reactjs.org/pull/2#pullrequestreview-201922125

File: TRANSLATION.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# reactjs.org saytining O'zbek tilidagi tarjimasi
+
+Hozirgi jarayon: 
+https://github.com/reactjs/uz.reactjs.org/issues/1
+
+## Tarjima bo'yicha kelishuv
+
+| Terminning asli | Tarjim |
```
Comment (rustamwin) [inline TRANSLATION.md:8]: *Tarjima

File: TRANSLATION.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+| ------------------ | ---------- |
+| API reference | API ma'lumotnoma |
+| array | massiv |
+| arrow function |  arrow funksiya, strelka funksiya  |
+| attribute | atribut |
+| batch | yangilash guruhi |
+| batching | guruhlash |
+| bug | bag, xatolik  |
```
Comment (rustamwin) [inline TRANSLATION.md:16]: What about `kamchilik` ?

File: TRANSLATION.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+| framework | freymvork |
+| function component | funksiyali komponent |
+| hook | ho'k, ilgak |
+| incapsulation | inkapsulyatsiya |
+| incapsulated | inkapsulyatsiya qilingan |
+| key | kalit |
+| lazy initialization | lezy boshlanish, dangasa boshlanish |
+| library | kutubxona |
+| lifecycle | hayotiy sikl |
+| lifecycle method | hayotiy jarayon metodi |
```
Comment (rustamwin) [inline TRANSLATION.md:37]: What about `hayotiy sikl usuli|uslubi`?

File: TRANSLATION.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+| paint | chizmoq, chizish, bo'yash |
+| prop | prop *(birlik)* |
+| props | props *(ko'plik)* |
+| production | prodakshin |
+| production mode | prodakshin rejim |
+| reuse | qayta ishlatish, qayta foydalanish |
+| React | React |
+| React element | React element |
+| reconciliation | kelishuv, muvofiqlashtirish |
+| render(ing) | chizmoq (chizish) |
```
Comment (rustamwin) [inline TRANSLATION.md:58]: What about `ko'rsatish` ?

File: TRANSLATION.md (hunk 1 above)
Comment (turdiyev) [reply TRANSLATION.md]: I just want to emphasize that Hook is Capitan Ho'k's name. 

O'zbek tilida: Kapitan Ho'k ni nomlanishi ya'ni qo'lidagi ilgak bilan nomlanganligini ta'kidlamoqchi edim. Ho'k uzbek tiliga ILGAK deb tarjima qilinadi.

**Comment by turdiyev** [comment]
> > I think you mean that add russian version. Ok, i will add russian column.
> 
> Nope
> 
> These two:
> [#2 (review)](https://github.com/reactjs/uz.reactjs.org/pull/2#pullrequestreview-201921988)
> [#2 (review)](https://github.com/reactjs/uz.reactjs.org/pull/2#pullrequestreview-201922125)

Done Umid. You should merge this PR, right?

File: TRANSLATION.md (hunk 1 above)
Comment (avocadowastaken) [reply TRANSLATION.md]: @turdiyev oh, I guess you can add note about this in hooks section https://reactjs.org/docs/hooks-intro.html

**Comment by avocadowastaken** [comment]
@turdiyev lgtm

**Comment by rustamwin** [comment]
@umidbekkarimov left a some suggestions

**Comment by avocadowastaken** [comment]
@rustamwin please discuss this with PR author - @turdiyev

File: TRANSLATION.md (hunk 3 above)
Comment (turdiyev) [reply TRANSLATION.md:8]: @rustamwin you can create PR for typo fixing.

File: TRANSLATION.md (hunk 6 above)
Comment (turdiyev) [reply TRANSLATION.md:58]: @rustamwin you can add your own variant.

File: TRANSLATION.md (hunk 5 above)
Comment (turdiyev) [reply TRANSLATION.md:37]: Yep, you can add them.

File: TRANSLATION.md (hunk 4 above)
Comment (turdiyev) [reply TRANSLATION.md:16]: hmm, maybe, but i think it not compatible variant.
Examples: 
- "there's a bug -> Kodingda kamchilik bor". 
- "It is a bug -> "bu kamchilik"

**Comment by turdiyev** [comment]
> @umidbekkarimov left a some suggestions

suggestions? for what? Could you please clarify what do you mean?

**Pull request merged**